projectile-rails.el

```diff
@@ -777,14 +780,21 @@ The mode of the output buffer will be `projectile-rails-compilation-mode'."
   "Generate a cache key based on the current directory and the given KEY."
   (format "%s-%s" default-directory key))
 
+(defun projectile-rails--rails-app-p (root)
+  "Returns t if any of the relative files in `projectile-rails-verify-root-files' is found.
+ROOT is used to expand the relative files."
+  (--any-p
+   (file-exists-p (expand-file-name it root))
+   (-list projectile-rails-verify-root-files)))
```

Comment:
why `-list` here?

Reply:
Sorry I should be using `define-obsolete-variable-alias` instead of `define-obsolete-variable`.

Reply:
So the `-list` is for the cases when someone customized `projectile-rails-verify-root-file`.

Reply:
I still don't get it, because you do it on `projectile-rails-verify-root-files` not `projectile-rails-verify-root-file`.

Is this for the case where someone had this:

``` elisp
;; Before
(setq projectile-rails-verify-root-file "foo")
;; Theoric after
(setq projectile-rails-verify-root-files '("foo"))
;; TLDR after
(setq projectile-rails-verify-root-files "foo")
```

And basically just added an `s` instead of also enclosing it in a list?

Reply:
Sorry for not being clear.
By using `-list` I want to bypass an issues that someone before this change did `(setq projectile-rails-verify-root-file "config/environment.rb")` in the init file. Because after this change, since the `projectile-rails-verify-root-file` will be an alias for `projectile-rails-verify-root-files`, we have no guarantee that it will be a list.
So a hypothetical situation:
```lisp
;; Someone's init file before the change
(setq projectile-rails-verify-root-file "config/environment.rb")

;; This change
(define-obsolete-variable-alias 'projectile-rails-verify-root-file 'projectile-rails-verify-root-files)

(defun projectile-rails--rails-app-p (root)
  (--any-p
   (file-exists-p (expand-file-name it root))
   projectile-rails-verify-root-files)) ;; <- might not be a list, and that would produce an error
```

WDYT? Is it a right approach, or do you have something different in mind?

Reply:
Ah, it didn't occur to me that `define-obsolete-variable-alias` would make it so `projectile-rails-verify-root-file` and `projectile-rails-verify-root-files` share the same address space. Okay then now I got it, of course your change makes total sense then.